crates/targets/aarch64_msvc/build.rs

```diff
@@ -1,6 +1,8 @@
 fn main() {
-    let target = std::env::var("TARGET").unwrap();
-    if target != "aarch64-pc-windows-msvc" && target != "aarch64-uwp-windows-msvc" {
```

Comment:
Your patch seems to be missing the UWP special case?

Reply:
Oh, wait `UWP` is a "vendor" so it'll work. Never mind then!

Reply:
🤔 I'm not seeing it, it's not checking for `uwp` because it doesn't check the vendor at all now, so all of `(pc, win7, uwp, rust9x)` would pass there.

Reply:
Ha, I got there eventually!